services/user/TokenSys/ui/src/App.tsx

```diff
@@ -77,6 +89,7 @@ function App() {
         try {
             await transfer(data);
             reset();
+            refetchData();
```

Comment:
Let's move this down to line 105 and replace (the now duplicate) `invalidateTransferHistoryQuery()` there. Better it refetches, even after a failure.